src/mobile/dropdown/test/className.test.ts

```diff
@@ -0,0 +1,46 @@
+import { expect, fixture } from "@open-wc/testing";
+import { MobileDropdown } from "../index";
+
+describe("MobileDropdown", () => {
+  describe("className", () => {
+    it("className default prop is null", async () => {
```

Comment:
@nmanhit 
Please remove `className` and change the `title` to the same content with expectation.
Ex: `it("should be set to empty string when initializing without props option")`